src/modules/daemonconfiguration/src/common.rs

```diff
@@ -11,6 +11,7 @@ pub enum MmiError {
     FailedRead,
     FailedAllocate,
     InvalidArgument,
+    PayloadSizeExceeded,
```

Comment:
>PayloadSizeExceeded, [](http://example.com/codeflow?start=4&length=20)

This is only valid (as an error) for MmiSet. MmiGet cannot return this error, instead, it must truncate payload.

Reply:
Changed